daemon/c_uevent.c

```diff
@@ -230,6 +230,7 @@ c_uevent_handle_event_cb(unsigned actions, uevent_event_t *event, void *data)
 			event_coldboot = uevent_event_replace_synth_uuid_new(event, "0");
 			if (!event_coldboot) {
 				ERROR("Failed to mask out container uuid from SYNTH_UUID in uevent");
+				uuid_free(synth_uuid);
```

Comment:
also in line 254?
=> may replace return by goto: error there to free other variables too?

Reply:
In other cases the other variables were not allocated. But an err jump label and goto err would be more readable. true. Will do this.